src/main/java/chatbot/Plana.java

```diff
@@ -0,0 +1,43 @@
+package chatbot;
+
+import java.io.IOException;
+
+import chatbot.exceptions.DukeException;
+
+/**
+ * Represents the main class responsible for running the chatbot 'Plana'. Critical components like UI, file IO are
+ * initialised here.
+ */
+public class Plana {
+    private final TaskList taskList;
+    private final Storage store;
+    private boolean shouldExit;
```

Comment:
Will it be better to name this isExit instead to follow the java coding standard?